eip/EIPS/eip-4337.md

```diff
@@ -145,9 +145,10 @@ interface IAggregatedAccount is IAccount {
 
 * **getAggregator()** returns the aggregator this account supports.
 * **validateUserOp()** (inherited from IAccount interface) MUST verify the `aggregator` parameter is valid and the same as `getAggregator`
-* The account should also support aggregator-specific getter (e.g. `getAggregationInfo()`).
+* The account should also support aggregator-specific getter.
```

Comment:
Maybe this is better left out of the ERC, like `the aggregator and the account interaction is not restricted by this ERC. The account may need to support aggregator-specific view functions.` or something along this line?